### ISSUE TITLE
Multi-interface support

### ISSUE THREAD

**Issue by philipcristiano** [mayuresh82/gocast#23]
I have a use case where a Nomad host will have containers binding to multiple interfaces (one physical, multiple VLAN interfaces). A single gocast instance will see all containers/advertised ports on a single Consul node "equally". The connection to the TOR will go over the default route / single connection. 

Would it be possible to make `enable_gocast` tag configurable to allow for multiple gocast instances running to each find separate groups of Consul services to advertise? The [`matchTag`](https://github.com/mayuresh82/gocast/blob/6f26e869645fe63ff37278ba8fe3d63b0498dda9/controller/consul.go#L20) looks in a good spot to pull from the agent config with other Consul config values.

**Comment by mayuresh82** [comment]
@philipcristiano  Yes, I have this staged in a branch as part of enabling ipv6 support. It still needs to be tested and will be a few weeks, but I can put this in a PR and if you would be able to test this for your use case, that would be great !

**Comment by philipcristiano** [comment]
@mayuresh82 that would be amazing! I'd love to test out a branch with that!
